db/convenience.cc

```diff
@@ -68,8 +68,8 @@ Status VerifySstFileChecksum(const Options& options,
       !kImmortal, false /* force_direct_prefetch */, -1 /* level */);
   reader_options.largest_seqno = largest_seqno;
   s = ioptions.table_factory->NewTableReader(
-      reader_options, std::move(file_reader), file_size, &table_reader,
```

Comment:
Thanks - fixed by renaming `FILE_READ_VERIFY_CHECKSUM_MICROS ` an related enum/string name to 
 `FILE_READ_DB_VERIFY_CHECKSUM_MICROS ` and adding sanitization to `VerifySstFileChecksum` so it does not update any stats by passing no IOActivity. It can be a follow-up to add a enum/stats about `VerifySstFileChecksum`